elastalert/config.py

```diff
@@ -151,6 +151,10 @@ def load_options(rule):
                                 'The index will be formatted like %s' % (token,
                                                                          datetime.datetime.now().strftime(rule.get('index'))))
 
+    # Kibana4 urls can't be generated based on host:port
```

Comment:
Or more precisely, Kibana4 host+port cannot be assumed to be the same as the ES index ElastAlert is querying.